install/class_source/class_LinkActivityDefinition_export.json

```diff
@@ -511,7 +510,6 @@
         "icon": null
     },
     "icon": "\/bundles\/pimcoreadmin\/img\/flat-color-icons\/sports_mode.svg",
-    "previewUrl": "\/__customermanagementframework\/object-preview\/link-activity-definition-preview",
```

Comment:
The preview for LinkActivityDefintion is not working anymore in Pimcore 11. We need to add a Preview Generator Class here.

Reply:
I created another PR for this https://github.com/pimcore/customer-data-framework/pull/494